cairo/tests/ethereum/cancun/test_state.py

```diff
@@ -116,6 +136,36 @@ def test_set_account(self, cairo_run, data, account: Optional[Account]):
         set_account(state, address, account)
         assert state_cairo == state
 
+    @given(
+        data=state_and_address_and_optional_key(), recipient_address=address, amount=...
+    )
+    def test_move_ether(
+        self, cairo_run, data, recipient_address: Address, amount: U256
+    ):
+        state, sender_address = data
+        # We need to create a deep copy of the state to avoid mutating the original state
+        # We can refactor it into a deep_copy State util if we ever need to do this again
+        python_state = _state_deep_copy(state)
```

Comment:
I do not understand why doing this
To avoid having doing a copy you first execute the cairo code, then the python one like other tests, unless I miss something?
Also if there is any need you can use https://docs.python.org/3/library/copy.html to copy objects if not mistaken

Reply:
yes, run the cairo operation before the python one.

Reply:
previously if i ran the cairo operation before the python one, then i'm not able to know which error will be thrown

Reply:
> I do not understand why doing this To avoid having doing a copy you first execute the cairo code, then the python one like other tests, unless I miss something? Also if there is any need you can use https://docs.python.org/3/library/copy.html to copy objects if not mistaken

you can't copy state because Account is a frozen type,  at least that's the error i was getting when trying to just use `deepcopy` from `copy` library